File: Getting Wood/gettingwood.c
#include <stdio.h>

int main() {
    char s[101];
    scanf("%[^\n]", s);

    for (int i = 0; s[i] != '\0'; i++) {
        if (s[i] == 't' && s[i + 1] == 'r' && s[i + 2] == 'e' && s[i + 3] == 'e') {
            printf("%d\n", i);
            return 0;
        }
    }

    printf("no trees here\n");
    return 0;
}

File: Getting Wood/gettingwood.py
s = input()

for i in range(len(s) - 3):
    if s[i] == "t" and s[i + 1] == "r" and s[i + 2] == "e" and s[i + 3] == "e":
        print(i)
        break
else:
    print("no trees here")
